### PR TITLE
[JSX] Refactor SearchableDropdown to not use document.querySelector

### PR DESCRIPTION
**Pull request by davidblader**
### Brief summary of changes
this refactors the SearchableDropdown to avoid reading directly from the DOM by relying on an additional prop value

### This resolves issue...
CCNA was having fatal JS errors on some forms due to reliance on document.querySelector

### To test this change...

- [ ] Make sure media's uploadForm still works, specifically for Site input

### CODE REVIEW

**Comment by davidblader** [comment]
@driusan @ridz1208  if possible i'd like to rebase this to be on bugfix, if that's OK / if there's a bugfix release planned before a minor release

File: jsx/Form.js [hunk 1]
```diff
@@ -168,6 +167,7 @@ class SearchableDropdown extends React.Component {
     if (!this.props.strictSearch && value === undefined) {
       value = e.target.value;
     }
+    this.props.onUserInput(this.props.name + 'Temp', e.target.value);
```
Comment (driusan) [inline jsx/Form.js]: I don't understand what this "Temp" thing is all about in this PR.

**Comment by driusan** [comment]
If it's a bug fix, I don't see any issue with it going to bugfix..

File: jsx/Form.js (hunk 1 above)
Comment (davidblader) [reply jsx/Form.js]: there needs to be an intermediary "temporary" value for the user's input to be stored in as they're typing. couldn't think of a better naming convention. open to any suggestions

**Pull request closed without merging**